pkg/api/http.go

```diff
@@ -256,9 +254,6 @@ func ParseSearchBlockRequest(r *http.Request) (*tempopb.SearchBlockRequest, erro
 	req.TotalRecords = uint32(totalRecords)
 
 	dataEncoding := r.URL.Query().Get(urlParamDataEncoding)
```

Comment:
It would fail with the error `"unknown encoding . Supported encodings v1 v2"` (the empty string is printed in the first sentence).   Yep, can add some comments.